scripts/find_unique_isoform.py

```diff
@@ -90,34 +97,71 @@ def readSeqsFunc(inBed, manifest):
         # config sequences
         for locus in outDict.keys():
 
-            outDict[locus][sample] = {}
-
+            checkList = []
             for read in samfile.fetch(locus[0], int(locus[1]), int(locus[2])):
-                qname = read.query_name.split('_')[2]
-                outDict[locus][sample][qname] = {}
-                outDict[locus][sample][qname]['seq'] = seqDict[qname]
-                outDict[locus][sample][qname]['start'] = read.reference_start
-                outDict[locus][sample][qname]['query_length'] = read.query_length
+
+                #flag = '0'*(12-len(format(read.flag, 'b'))) + format(read.flag, 'b')
+                #if read.mapping_quality < 60: continue # filter by mapping quality 60
+                #if flag[0] == 1: continue # ignore supplementary alignment
+                #if flag[3] == 1: continue # ignore non-primary alignment
+
+                temp = isoform()
+                temp.sample = sample
+                temp.name = read.query_name.split('_')[2]
+                temp.id = read.query_name
+                if temp.id in checkList: continue
```

Comment:
Wouldn't it be better to do `if temp.id not in checkList:`? 

`continue` always confuses me when used this way in python @BidaGu